package.json

```diff
@@ -79,16 +79,17 @@
     "hapi": "18.1.0",
     "hapi-auth-basic": "5.0.0",
     "hapi-auth-bearer-token": "6.1.1",
-    "hapi-swagger": "9.3.1",
+    "hapi-swagger": "9.4.2",
     "immutable": "3.8.2",
     "inert": "5.1.2",
     "joi": "14.3.1",
     "joi-currency-code": "1.2.1",
     "joi-date-extensions": "1.2.0",
-    "jsdoc": "3.5.5",
+    "jsdoc": "3.6.2",
     "knex": "0.16.3",
     "lodash": "4.17.11",
     "mustache": "3.0.1",
+    "raw-body": "2.4.0",
```

Comment:
any reason you are importing this?

Reply:
the plugin uses it to get the raw body from the stream

Reply:
he uses it in src/lib/hapi/plugins/rawPayloadToDataUri.js